### PR TITLE
Updating the INSPIRE overview file and adding the INSPIRE screenshot

### PR DESCRIPTION
**Pull request by df-git**

### CODE REVIEW

**Comment by df-git** [comment]
The checks are failing because there is a reference to the re3gistry_overview file that has just been added and it needs to be approved too.

**Comment by cvvergara** [comment]
I see this one:
```
2020-09-24T15:26:31.5794726Z (line   39) broken    https://etf-validator.net - 503 Server Error: Service Temporarily Unavailable for url: https://etf-validator.net/
```
here:
https://github.com/OSGeo/OSGeoLive-doc/runs/1161252637?check_suite_focus=true#step:9:574

the complete section of linkcheck is:
```
ok        https://inspire.ec.europa.eu/about-inspire/563
(line   17) ok        https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32007L0002&from=EN
(line   17) redirect  https://www.iso.org - with Found to https://www.iso.org/home.html
(line   19) redirect  https://inspire.ec.europa.eu/Themes - permanently to https://inspire.ec.europa.eu/Themes/Data-Specifications/2892
(line   33) ok        https://wiki.osgeo.org/wiki/INSPIRE
(line   36) ok        https://ec.europa.eu/jrc/en
(line   39) broken    https://etf-validator.net - 503 Server Error: Service Temporarily Unavailable for url: https://etf-validator.net/
(line   66) -local-   ../sponsors_osgeo.html
(line   71) -local-   ../sponsors_osgeo.html
(line   76) -local-   ../sponsors_osgeo.html
(line   39) ok        https://inspire.ec.europa.eu/validator/about/
(line   40) ok        https://inspire-geoportal.ec.europa.eu/
(line   38) ok        https://inspire.ec.europa.eu/registry
(line   25) ok        https://inspire.ec.europa.eu/Data-Models/Data-Specifications/2892
```
This is not checked because it does not have the `http://` or `https://` so the link checker does not consider it.
```
:doc:`Re3gistry <re3gistry_overview>`

I did a double check downloading in my computer and there was no problem, so I will re-run the link tests

File: doc/overview/inspire_overview.rst [hunk 1]
```diff
@@ -9,40 +11,33 @@
...
+
+The datasets falling under the scope of the INSPIRE Directive are grouped into 34 cross-sectoral categories names `spatial data themes <https://inspire.ec.europa.eu/Themes>`__, which are defined in the Annexes of the Directive and reflect two main types of data: geospatial reference data, presented in Annex I and partly Annex II, which define a location reference that the remaining domain-specific themes (in Annex III and partly Annex II) can then refer to.
```
Comment (cvvergara) [inline doc/overview/inspire_overview.rst:16]: although we dont have yet set permanent redirections warnings to be marked as errors we have that plan in the future, as it means that the original page no longer exists. Can you please change
https://inspire.ec.europa.eu/Themes 
to 
https://inspire.ec.europa.eu/Themes/Data-Specifications/2892

**Review by cvvergara — CHANGES_REQUESTED** [review]

**Comment by cvvergara** [comment]
I run twice in my computer:
```
writing output... [ 22%] overview/inspire_overview                                                                                                                                                                               
ok        https://inspire.ec.europa.eu/about-inspire/563
(line   25) ok        https://inspire.ec.europa.eu/Data-Models/Data-Specifications/2892
(line   19) redirect  https://inspire.ec.europa.eu/Themes - permanently to https://inspire.ec.europa.eu/Themes/Data-Specifications/2892
(line   33) ok        https://wiki.osgeo.org/wiki/INSPIRE
(line   17) ok        https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32007L0002&from=EN
(line   17) redirect  https://www.iso.org - with Found to https://www.iso.org/home.html
(line   39) ok        https://etf-validator.net
(line   66) -local-   ../sponsors_osgeo.html
(line   71) -local-   ../sponsors_osgeo.html
(line   76) -local-   ../sponsors_osgeo.html
(line   36) ok        https://ec.europa.eu/jrc/en
(line   39) ok        https://inspire.ec.europa.eu/validator/about/
(line   40) ok        https://inspire-geoportal.ec.europa.eu/
(line   38) ok        https://inspire.ec.europa.eu/registry
```
And I dont have problems with the link, https://etf-validator.net
But all the time I ran the actions that link shows:
```
broken    https://etf-validator.net - 503 Server Error: Service Temporarily Unavailable for url: https://etf-validator.net/
```
So, its not the only case that issue happens, can you please edit this:
https://github.com/OSGeo/OSGeoLive-doc/blob/master/doc/conf.py#L266
Add the  https://etf-validator.net  link, where its at, and the error it shows.

File: doc/overview/inspire_overview.rst (hunk 1 above)
Comment (cvvergara) [reply doc/overview/inspire_overview.rst:16]: I dont know why this is shown 2 times

**Comment by cvvergara** [comment]
A preview of how it looks
image

**Review by df-git — COMMENTED** [review]
The changes are fine

**Comment by cvvergara** [comment]
@df-git there is a missing comma at the end of the line of the conf.py

**Comment by cvvergara** [comment]
This PR can be merged after #621 because it references a project not yet included on the disk

**Review by cvvergara — APPROVED** [review]

**Pull request merged**